pallets/phala-world/src/nft_sale.rs

```diff
@@ -1572,20 +1573,20 @@ where
 		pallet_uniques::Pallet::<T>::freeze(
 			Origin::<T>::Signed(overlord).into(),
 			origin_of_shell_collection_id,
-			nft_id,
+			nft_id.into(),
 		)?;
 
 		Self::deposit_event(Event::OriginOfShellMinted {
 			rarity_type,
-			collection_id: origin_of_shell_collection_id,
+			collection_id: origin_of_shell_collection_id.into(),
 			nft_id,
```

Comment:
I see most have `nft_id.into()` but others do not. Do they all need to have `.into()`?

Reply:
Basically PW pallet use `CollectionId` and `NftId` from `rmrk-primitives`, which are not traits type `CollectionId` and `ItemId`, so it has to convert.

I do not change PW codes too much, only add `into()` when the compiler says error, but I may over-done this.